persistence/src/commonMain/kotlin/com/wire/kalium/persistence/dao/message/MessageDAOImpl.kt

```diff
@@ -232,6 +235,25 @@ internal class MessageDAOImpl internal constructor(
         queries.selectById(id, conversationId, mapper::toEntityMessageFromView).executeAsOneOrNull()
     }
 
+    override suspend fun getMessageAssets(
+        conversationId: QualifiedIDEntity,
+        limit: Int,
+        offset: Int
+    ): List<AssetMessageEntity> =
+        withContext(coroutineContext) {
+            assetViewQueries.getAssetMessagesByConversationIdAndMimeTypes(
+                conversationId,
+                listOf(MessageEntity.Visibility.VISIBLE),
+                listOf(MessageEntity.ContentType.ASSET),
+                listOf(
+                    "image/jpg", "image/jpeg", "image/png", "image/gif", "image/webp"
```

Comment:
Can we have this in a constant?

Reply:
Also will this only handle images right? Is there any plan or will there be another query or something for videos and others as well?

Reply:
Yes, there will be another query for files but I will use MesssageView for it because it's very similar to normal messages